src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java

```diff
@@ -366,7 +371,14 @@ private void checkExcludedDates(Appeal appeal) {
 
     private boolean isPhoneNumberValid(Appellant appellant) {
         if (appellant != null && appellant.getContact() != null && appellant.getContact().getPhone() != null) {
-            return appellant.getContact().getPhone().matches("^[0-9\\-+ ]{10,17}$");
+            return PhoneNumbersUtil.isValidLandLineNumber(appellant.getContact().getPhone());
```

Comment:
Have you checked this changed validation works with Robotics? The previous regex was copied from the robotics schema. If the edge cases(<10 or >17) still work then fine

Reply:
I've checked the PhoneNumberUtil for landline is not accepting more than 10. so revert it back as per Robotics validation.
For Mobile validation using this PhoneNumbersUtil